ChimpAndSee/index.html

```diff
@@ -22,8 +22,13 @@
         <link rel="icon" href="favicon.ico" type="image/ico"/>
 
         <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+        <script src="js/main.js"></script>
+
+        <script>
+          window.onload = checkPath;
```

Comment:
I think you'll have to move the checkPath function into this script tag too or else this will be undefined.

Reply:
That shouldn't be a problem. `checkPath` here is a global variable.

Reply:
AH, I missed that main.js is loading before this.

Reply:
If it's declared as a function in main.js, then it'll be defined at runtime, even if it's declared after this script tag.